src/components/algolia-search/algolia-search.module.css

```diff
@@ -0,0 +1,148 @@
+.root {
+  position: relative;
+  margin-bottom: 34px;
+}
+
+.form {
+  border: none;
+  box-shadow: var(--token-surface-inset-box-shadow);
+  border-radius: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+
+  &:focus-within {
+    outline: none;
+
+    /* --token-color-focus-action-internal */
+
+    /* border: 1px solid var(--token-color-palette-blue-300); */
+    box-shadow: var(--token-focus-ring-action-box-shadow);
+  }
+}
+
+.keyboardHintIcon {
+  color: var(--token-color-foreground-faint);
+  position: absolute;
+  top: calc(50% - 8px);
+  right: 12px;
+  font-size: 10px;
+  font-weight: 600;
+  height: 16px;
+  width: 16px;
+  line-height: 16px;
+}
+
+.inputWrapperPrefix {
+  display: inline-block;
+}
+
+.submitButton {
+  border: 0;
+  background: transparent;
+  height: 100%;
+  display: flex;
+  border-radius: 5px;
+  flex-direction: row;
+  align-items: center;
+
+  & svg {
+    color: var(--token-color-foreground-faint);
+  }
+}
+
+.input {
+  flex-grow: 1;
+  position: relative;
+  border: 0;
+
+  /* TODO: token for font-size? */
```

Comment:
Keeping the font-size at 16 here to avoid iOS automatically zooming on focus.

 (ref: https://css-tricks.com/16px-or-larger-text-prevents-ios-form-zoom)